### PR TITLE
Add @vladikoff’s post on PWA + desktop

### PR DESCRIPTION
**Pull request by mathiasbynens**
@operasoftware/odevrel

### CODE REVIEW

**Comment by pepelsbey** [comment]
Do we really need a `native-pwa.jpg` blurry photo of a screen? It doesn’t look good or valueable. I’d rather add a proper screen shot instead.

**Comment by vladikoff** [comment]
> Do we really need a native-pwa.jpg blurry photo of a screen? It doesn’t look good or valueable. I’d rather add a proper screen shot instead.

Sure! It was more of an intro photo but I can provide a better one (or reuse from the article: )

File: src/articles/_posts/2016-07-20-pwa-desktop.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+- vlad-filippov
+intro: 'Progressive Web Apps are getting ready for desktop.'
+tags:
+- pwa
+license: cc-by-3.0
+published: false
+---
+
+<figure block="figure">
+	<img elem="media" src="{{ page.id }}/native-pwa.jpg" alt="Progressive Web Apps running as separate native apps on OS X">
```
Comment (vladikoff) [inline src/articles/_posts/2016-07-20-pwa-desktop.md]: @mathiasbynens  Wanna change this to: `Progressive Web Apps running as separate native apps` and remove`native-pwa.jpg` from PR per feedback?

Comment (mathiasbynens) [inline src/articles/_posts/2016-07-20-pwa-desktop.md]: Let’s keep `multi-platform.jpg` where it is. I’ve removed `native-pwa.jpg` now.

**Comment by mathiasbynens** [comment]
I’ve removed the intro image.

**Comment by vladikoff** [comment]
> I’ve removed the intro image.

Thanks!! 👍

**Comment by andreasbovens** [comment]
Looks good to me!

**Comment by pepelsbey** [comment]
Will merge and publish it today

**Pull request merged**